client/my-sites/importer/importer-header.jsx

```diff
@@ -102,3 +114,9 @@ export default React.createClass( {
 		);
 	}
 } );
+
+const mapDispatchToProps = dispatch => ( {
+	startImport: flowRight( dispatch, startImport )
+} );
+
+export default connectDispatcher( null, mapDispatchToProps )( ImporterHeader );
```

Comment:
mapDispatchToProps can also be an object:
```javascript
export default connectDispatcher( null, { startImport } )( ImporterHeader );
```
https://github.com/reactjs/react-redux/blob/master/docs/api.md#arguments